src/js/types/generic.type.ts

```diff
@@ -22,6 +22,10 @@ export type DrmConfig = GenericObject;
 
 export type PluginObj = GenericObject;
 
+export type PlaylistItem = GenericObject;
```

Comment:
In model.ts, I just did `import type PlaylistItem from 'playlist/item';`. For js imports this type is treated as `any` (unless TS can infer some typing from es6 classes). So, it's basically the same thing and we don't need to make a change later.

Reply:
hmmm vs code was throwing an error. will update if possible